test/ngAnimate/animateSpec.js

```diff
@@ -292,6 +292,129 @@ describe("animations", function() {
       });
     });
 
+    describe('customFilter()', function() {
+      it('should clear the `customFilter` if no function is passed',
+        module(function($animateProvider) {
+          expect($animateProvider.customFilter()).toBeNull();
+
+          $animateProvider.customFilter(angular.noop);
+          expect($animateProvider.customFilter()).toEqual(jasmine.any(Function));
+
+          $animateProvider.customFilter({});
```

Comment:
I missed that this needed to land in 1.6.0 (due to the BC) :(
I will make only `customFilter` throw for 1.6.x/1.7.x and change `classNameFilter` to also throw for 1.7.x only.